runtime/src/lib.rs

```diff
@@ -642,6 +630,49 @@ impl pallet_session::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub IntergalacticAccounts: Vec<AccountId> = vec![
+		hex_literal::hex!["0abad795adcb5dee45d29528005b1f78d55fc170844babde88df84016c6cd14d"].into(),
+	];
+}
+
+pub struct EnsureIntergalactic;
+impl EnsureOrigin<Origin> for EnsureIntergalactic {
+	type Success = AccountId;
+
+	fn try_origin(o: Origin) -> Result<Self::Success, Origin> {
+		Into::<Result<RawOrigin<AccountId>, Origin>>::into(o).and_then(|o| match o {
+			RawOrigin::Signed(caller) => {
+				if IntergalacticAccounts::get().contains(&caller) {
+					Ok(caller)
+				} else {
+					Err(Origin::from(Some(caller)))
+				}
+			}
+			r => Err(Origin::from(r)),
+		})
+	}
+
+	#[cfg(feature = "runtime-benchmarks")]
+	fn successful_origin() -> Origin {
+		Origin::from(RawOrigin::Signed(Default::default()))
+	}
+}
+
+parameter_types! {
+	pub MinVestedTransfer: Balance = BSX / 100_000;
+	pub const MaxVestingSchedules: u32 = 100;
+}
+
+impl orml_vesting::Config for Runtime {
+	type Event = Event;
+	type Currency = Balances;
+	type MinVestedTransfer = MinVestedTransfer;
+	type VestedTransferOrigin = EnsureIntergalactic;
+	type WeightInfo = ();
+	//type MaxVestingSchedules = MaxVestingSchedules;
```

Comment:
```suggestion
	type MaxVestingSchedules = MaxVestingSchedules;
```

is this enforced or is it only used for benchmarks?

Reply:
this type was added in 0.4.1-dev